### PR TITLE
Port to GTK+ 3 and support Python 3

### PR DESCRIPTION
**Pull request by kabili207**
Support for both was added by using pygtkcompat. Minimal changes
were required to allow this. It will still run correctly under Python 2. 

The source code was also run through a formatter to correct the
heavy mix of tabs and spaces. I have no idea how it even ran with
such inconsistent indentation.

This should also address issue #1

### CODE REVIEW

**Comment by kernelOfTruth** [comment]
@kabili207 

I actually ran into run-issues while doing some changes to the file so 

>The source code was also run through a formatter to correct the
heavy mix of tabs and spaces. I have no idea how it even ran with
such inconsistent indentation.

yeah :)


Thanks a lot for your effort !

During the next days I won't have a chance to boot into my Linux system (main OS) to review it in-depth or run it,

but from a quick glance at it - it looks great - much improved (github preview is on a strike)

so - approved 😄 

You're using the equalizer a lot ?

I don't want to miss out on it anymore since it really improves the sound output.

Let's hope the github web front-end for merging preserves the authorship this time ...

**Comment by kernelOfTruth** [comment]
Oh great, it worked 😀 

it didn't with ZFSOnLinux before, so I was a bit hesitant ...

**Comment by kabili207** [comment]
>You're using the equalizer a lot ?

I've been using the older FFT equalizer, but I got fed up all of its problems and decided to switch to the LADSPA equalizer. I eventually found your repo here when I couldn't find an ebuild in portage.

>Thanks a lot for your effort !

There wasn't too much effort involved, actually. It just looks like I did because the formatter did a lot. All I really did was add some parentheses to the print statements and toss the compat stuff at the top.

I'm going to see if I can make an ebuild for this over the weekend. Can you bump the version number on this and create a release for it? It'll make things much easier.

**Comment by kernelOfTruth** [comment]
@kabili207 sorry for the delay,

I hardly used my Linux system within the last 3 weeks.

Is the following similar to what you had in mind ?

https://github.com/kernelOfTruth/pulseaudio-equalizer/commit/6bbd4b3e10b7e39fe591b2db62cbf6195d349dcc

**Comment by kabili207** [comment]
I was thinking more along the lines of this: https://help.github.com/articles/creating-releases/

**Comment by kernelOfTruth** [comment]
@kabili207 github makes it rather convenient, thanks for the link :)

https://github.com/kernelOfTruth/pulseaudio-equalizer/releases/tag/v3.0

**Comment by kabili207** [comment]
Okay, so I may be about six months late, but I've finally created an ebuild for this. You'll need to pull it from the zyrenth overlay in layman.

http://gpo.zugaina.org/media-sound/pulseaudio-equalizer
